src/views/settings/p2p-stats/p2p-stats.module.css

```
@@ -0,0 +1,197 @@
+  padding: 2px 4px 3px 0;
+  vertical-align: top;
+}
+
+.statName {
+  width: 34%;
+  white-space: nowrap;
+  font-weight: bold;
+}
+
+.statValue {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)